src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,265 @@
+from py42.clients.util import get_all_pages
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    Administrator utility to manage High Risk employees.
+    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
+
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
+    # if we want to refactor version and prefix to a common configuration.
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(HighRiskEmployeeClient, self).__init__(session)
+        self._user_context = user_context
+
+    def _add_high_risk_employee(self, tenant_id, user_id):
+
+        data = {"tenantId": tenant_id, "userId": user_id}
+        resource = u"/highriskemployee/add"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def add(self, user_id=None):
+        """
+        Adds a user to high risk employee lens.
+
+        If a user is already added to high risk employee lens, further attempts to add will
+        return failure error.
+
+        Args:
+            user_id (str/int): Id of the user who needs to be added to HRE lens.
+
+        Returns:
+            :class:`py42.sdk.response.Py42Response`
+        """
+
+        tenant_id = self._user_context.get_current_tenant_id()
+        return self._add_high_risk_employee(tenant_id, user_id)
+
+    def set_alerts_enabled(self, enabled=True):
+        """
+        Enable alerts.
+        #TODO Confirm
+        # `toggle_alerts` would be a better method name, if we intend to allow to disable
+        # alerts as well.
+        Args:
+            enabled (bool): Whether to enable alerts for all users
+
+        Returns:
+            :class:`py42.sdk.response.Py42Response`
+        """
+        data = {
+            "tenantId": self._user_context.get_current_tenant_id(),
+            "alertsEnabled": enabled,
+        }
+        resource = u"/highriskemployee/setalertstate"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def remove(self, user_id):
+        """
+        Remove an user from high risk employee lens.
+
+        Args:
+            user_id (str/int): Id of the user who needs to be added to HRE lens.
+
+        Returns:
+            :class:`py42.sdk.response.Py42Response`
+        """
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/highriskemployee/remove"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def get(self, user_id):
+        """
+        Get user information.
+
+        Args:
+            user_id (str/int): Id of the user who needs to be added to HRE lens.
```

Comment:
id of user who _has been added_ to the High Risk Employee detection list.

Reply:
Changed.